src/redux/book/book.js

```diff
@@ -1,41 +1,57 @@
-// action
-import { REMOVE_BOOK, ADD_BOOK } from '../../util/constant';
-
-const initialState = [
-  {
-    title: 'book 1',
-    author: 'author 1',
-    id: 1,
-  },
-  {
-    title: 'book 2',
-    author: 'author 2',
-    id: 2,
-  },
-  {
-    title: 'book 3',
-    author: 'author 3',
-    id: 3,
-  },
-];
-
-// reducer
-
-export default (state = initialState, action) => {
+import { getData, removeData, uploadData } from '../../util/axios';
+// Actions
```

Comment:
- [ ] `createAsyncThunk` should be used in the project for creating thunk action creators, kindly implement this it's a requirement